README-DEPLOY.md

```diff
@@ -128,6 +128,29 @@ You can find the open-source project and the instructions to deploy it here: [ma
 
 ![Power Tuner UI](https://github.com/mattymoomoo/aws-power-tuner-ui/blob/master/imgs/website.png?raw=true)
 
+## Option 6: deploy the SAR app with Terraform
+
+Simply add the `aws_serverlessapplicationrepository_cloudformation_stack` resource below to your Terraform code and deploy as usual through `terraform apply`.
+
+```hcl
+resource "aws_serverlessapplicationrepository_cloudformation_stack" "lambda-power-tuning" {
+  name           = "lambda-power-tuner"
+  application_id = "arn:aws:serverlessrepo:us-east-1:451282441545:applications/aws-lambda-power-tuning"
+  capabilities   = ["CAPABILITY_IAM"]
+
+  parameters = {
+    # Many of these parameters are optional and are only shown here for demonstration purposes
```

Comment:
They're all optional :) Maybe we could comment the whole parameters section?

I'm doing the same here: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/scripts/deploy-sar-app.yml

Reply:
Good idea. I commented them out and updated to include a link to that page with all the params that you provided.